src/main/java/com/gooddata/GoodDataSettings.java

```diff
@@ -22,6 +24,7 @@
     private int connectionTimeout = secondsToMillis(10);
     private int connectionRequestTimeout = secondsToMillis(10);
     private int socketTimeout = secondsToMillis(60);
+    private String userAgent = StringUtils.EMPTY;
```

Comment:
Why not to use string- it's the classic billion dollar mistake problem... And why to use constant... In this place it doesn't matter. But otherwise because of maintainability. And if you use empty string on multiple places in code (which can happen), Sonar will complain about it. I can put there null or "", I don't care.